src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -618,6 +619,11 @@ namespace OpenLoco::ObjectManager
         return obj;
     }
 
+    bool isTemporyObjectLoad()
```

Comment:
```suggestion
    bool isTemporaryObjectLoaded()
```

Reply:
Its not actually that. Its whether the temporary object load function is being called. The temp object isn't loaded at this point just half way through it.

Reply:
Okay, in that case, let's just address the typo:
```suggestion
    bool isTemporaryObjectLoad()
```